README.md

```
@@ -70,4 +121,4 @@ We are maintaining the future roadmap using the [issues](https://github.com/true
 
 ## Star History
 
-[![Star History Chart](https://api.star-history.com/svg?repos=truefoundry/CruiseKube&type=Date)](https://www.star-history.com/#truefoundry/CruiseKube&Date)
+[![Star History Chart](https://api.star-history.com/svg?repos=truefoundry/CruiseKube&type=Date)](https://www.star-history.com/#truefoundry/CruiseKube&Date)
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)